apache-activemq-mixin/dashboards/apache-activemq-instance-overview.libsonnet

```diff
@@ -0,0 +1,1101 @@
+        insertNulls: false,
+        lineInterpolation: 'smooth',
+        lineWidth: 2,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'never',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+          },
+        ],
+      },
+      unit: 's',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'right',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'multi',
+      sort: 'desc',
+    },
+  },
+};
+
+local garbageCollectionCountPanel(matcher) = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'increase(java_lang_g1_young_generation_collectioncount{' + matcher + ', instance=~"$instance"}[$__interval:])',
+      datasource=promDatasource,
+      legendFormat='{{activemq_cluster}} - {{instance}}',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Garbage collection count / $__interval',
+  description: 'The recent increase in the number of garbage collection events for the JVM.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        axisShow: false,
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 25,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        insertNulls: false,
+        lineInterpolation: 'smooth',
+        lineWidth: 2,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'never',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+          },
+        ],
+      },
+      unit: 'none',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'right',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'multi',
+      sort: 'desc',
+    },
+  },
+};
+
+local activemqAlertsPanel(matcher) = {
+  datasource: promDatasource,
+  targets: [],
+  type: 'alertlist',
+  title: 'ActiveMQ alerts',
+  description: 'Firing alerts for Apache ActiveMQ environment.',
+  options: {
+    alertName: '',
+    dashboardAlerts: false,
+    alertInstanceLabelFilter: '{' + matcher + ', instance=~"${instance:regex}"}',
```

Comment:
This one is tricky. it would only work if all of selectors are in form ${variable:regex}. And here I think matcher would expand to $variable syntax.

Reply:
Yea this one was tricky, i made another variable that will take care of this and expand correctly into the new syntax. This should be good to go!